packages/sdk-react/src/components/payables/PayableDetails/PayableDetailsForm/helpers.ts

```diff
@@ -225,3 +226,30 @@ function formatTaxToMinorUnits(tax: number): number {
 function formatTaxFromMinorUnits(tax: number): number {
   return tax / 100;
 }
+
+export const isFieldRequired = (
+  fieldName: string,
+  ocrRequiredFields: Record<string, boolean> | undefined,
+  value?: string | null
+) => {
```

Comment:
I would add some generics on it to have strictly typed `fieldName` and `ocrRequiredFields`